packages/pwa/app/utils/url.test.js

```diff
@@ -185,3 +195,31 @@ describe('removeQueryParamsFromPath test', () => {
         expect(updatedUrl).toEqual('/en/product/25501032M?something=123')
     })
 })
+
+describe('routeBuilder test', () => {
```

Comment:
I think these are outdated, removing them will make reviewing a tiny bit easier :)

Reply:
good catch, I've removed it